app/pages/patientnew/patientnew.js

```diff
@@ -70,6 +71,37 @@ export let PatientNew = React.createClass({
       name: 'diagnosisDate',
       label: 'Diagnosis date',
       type: 'datepicker'
+    },
+    {
+      name: 'dataDonate',
+      label: 'Donate your anonymized data',
+      type: 'checkbox'
+    },
+    {
+      name: 'dataDonateDestination',
+      type: 'select',
+      value: '',
+      placeholder: '',
+      items: [
+        {value: '', label: 'Share Proceeds with', 'disabled':'true'},
+        {value: 'BT1', label: 'Beyond Type 1'},
+        {value: 'CWD', label: 'Children with Diabetes'},
+        {value: 'CDN', label: 'College Diabetes Network'},
+        {value: 'DHF', label: 'Diabetes Hands Foundation'},
+        {value: 'JDRF', label: 'JDRF'},
+        {value: 'DIATRIBE', label: 'The Diatribe Foundation'},
+        {value: 'NSF', label: 'Nightscout Foundation'},
+        {value: 'T1DX', label: 'T1D Exchange'}
+      ]
+    },
+    {
+      name: 'donateExplainer',
+      type: 'explanation',
+      text: (<div style={{'textAlign':'left'}}>
+        By securely and anonymously donating your diabetes data you are helping
+        researchers, device makers, and other innovators. Read everything about
+        this project <a href="http://tidepool.org/blog">here</a>.
```

Comment:
It will. The post isn't up yet and, thus, the URL isn't finalized. Likely will come in sometime tomorrow, but the text change should be easy to get through without much fuss.